app/models/manageiq/providers/ansible_playbook_workflow.rb

```diff
@@ -16,6 +16,12 @@ def launch_runner
     Ansible::Runner.run_async(env_vars, extra_vars, playbook_path, kwargs)
   end
 
+  def process_abort(*args)
```

Comment:
oh nice!  I didn't realize process_abort was part of the ansible_playbook_workflow...I thought it was something on the service side.

Reply:
@lfu Can you move this into the base class?  Role will have to do the exact same thing, so this should be in the base runner class.